dali/operators/random/uniform.cc

```diff
@@ -19,12 +19,20 @@ namespace dali {
 
 void Uniform::AssignRange(HostWorkspace &ws) {
   auto &output = ws.OutputRef<CPUBackend>(0);
+  if (range_[0] >= range_[1]) {
+    for (int i = 0; i < batch_size_; ++i) {
+      sample_data[k] = range_[0];
+    }
+    return;
+  }
```

Comment:
I suggest adding argument verification in `Uniform` constructor, sth like:
```c++
    if (discrete_mode_) {
      set_ = spec.GetRepeatedArgument<float>("values");
      DALI_ENFORCE(!set_.empty(), "`values` argument cannot be empty");
    } else {
      range_ = spec.GetRepeatedArgument<float>("range");
      DALI_ENFORCE(range_.size() == 2, "`range` argument shall contain precisely 2 values");
      DALI_ENFORCE(range_[0] < range_[1],
                   "Invalid range. It shall be left-closed [a, b), where a < b");
    }
```

Reply:
Done